src/fusion_power_plant_tests.cc

```diff
@@ -37,18 +64,442 @@ TEST_F(FusionPowerPlantTest, Print) {
   // Test FusionPowerPlant specific aspects of the print method here
 }
 
-// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(FusionPowerPlantTest, Tick) {
-  ASSERT_NO_THROW(facility->Tick());
-  // Test FusionPowerPlant specific behaviors of the Tick function here
-}
-
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 TEST_F(FusionPowerPlantTest, Tock) {
   EXPECT_NO_THROW(facility->Tock());
   // Test FusionPowerPlant specific behaviors of the Tock function here
 }
 
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+TEST_F(FusionPowerPlantTest, BlanketCycle) {
+  // Test that the agent correctly removes and replaces a portion of the
+  // blanket every blanket turnover period. The default period is 1 timestep
+  // so it is left undefined here.
+
+  std::string config =
+      " <fusion_power>300</fusion_power>"
+      "  <TBR>1.00</TBR>"
+      " <reserve_inventory>6.0</reserve_inventory>"
+      " <sequestered_equilibrium>2.121</sequestered_equilibrium>"
+      " <fuel_incommod>Tritium</fuel_incommod>"
+      " <blanket_incommod>Enriched_Lithium</blanket_incommod>"
+      " <blanket_outcommod>Depleted_Lithium</blanket_outcommod>"
+      " <blanket_inrecipe>enriched_lithium</blanket_inrecipe>"
+      " <blanket_size>1000</blanket_size>"
+      " <blanket_turnover_fraction>0.03</blanket_turnover_fraction>"
+      " <he3_outcommod>Helium_3</he3_outcommod>";
```

Comment:
All of your sims use essentially this config - maybe it can be defined once globally?  If you want to include the case with the wrong fuel, set the `fuel_incommod` to be `Fuel` and define the source correctly (with tritium) in most cases and incorrectly (with enriched lithium) in that case.

You can also append the extra line for `Li7_contribution` in those cases.

Reply:
I did this in sort of a "dumb" way to avoid the overhead that comes with trying to be clever. I think it achieves the spirit of what you were asking for, but let me know if you want me to go the extra step.